### ISSUE TITLE
feature: integrity missing

### ISSUE THREAD

**Issue by BananaAcid** [Jakobud/vscode-cdnjs#12]
It would be great (since Safari is very keen in having it for remote urls) to be able to insert selection as tag and have an integrity attribute added. Like using https://www.srihash.org/

**Comment by Jakobud** [comment]
Yes, this has always simply been a limitation of the CDNJS API. It has been added to their API:

https://github.com/cdnjs/new-website/pull/246#issuecomment-580266948

But it not yet deployed to production yet. Whenever they get this in I will update the extension accordingly.

Thanks!

**Comment by Jakobud** [comment]
I see that thanks. I will work on adding this soon.

**Comment by BananaAcid** [comment]
All my students students use the extension in der projects for their finals in may. ;)

**Comment by BananaAcid** [comment]
https://github.com/cdnjs/new-website/pull/246#issuecomment-620740071
